src/components/LoginForm/index.js

```diff
@@ -9,15 +9,32 @@ import useForm from 'hooks/useForm';
 import useValidation from 'hooks/useValidation';
 import loginValidations from 'validations/loginValidations';
 import ErrorView from 'components/common/ErrorView';
+import Button from 'components/common/Button';
 import useTextInputProps from 'hooks/useTextInputProps';
 import strings from 'locale';
+import { BLACK, ERROR } from 'constants/colors';
 import styles from './styles';
 
 const FIELDS = {
   email: 'email',
   password: 'password',
 };
 
+const BUTTONPROPS = {
+  width: 114,
+  backgroundColor: BLACK,
+};
+
+const INPUTBORDER = {
+  borderColor: BLACK,
+  borderWidth: 0.5,
+};
+
+const INPUTBORDERERROR = {
+  borderColor: ERROR,
+  borderWidth: 1.5,
+};
```

Comment:
What about moving this to a new file and creating them usine react antive stylesheet?